libs/libc/unistd/Kconfig

```diff
@@ -111,7 +111,7 @@ config LIBC_HOSTNAME
 	---help---
 		A unique name to identify device on the network
 
-config LIBC_OPEN_MAX
+config OPEN_MAX
```

Comment:
can we keep LIBC_ prefix? since all libc option start with it.

Reply:
let revert this change, just move NAME_MAX/LINE_MAX/PATH_MAX to libc